omero_figure/static/figure/css/figure.css

```diff
@@ -711,7 +711,7 @@
         display: inline-block;
     }
 
-    .lutBg {
+    .lutBg, .ch_slider .ui-slider-range {
         /* NB: when updating png, consider using different name to avoid cache */
         background-image: url('../images/luts_10.png');   /* each lut is 10px high */
```

Comment:
own copy
this is the "out of synch" I am referring to in https://github.com/openmicroscopy/openmicroscopy/pull/5210